src/components/BoardItem.tsx

```diff
@@ -0,0 +1,91 @@
+import styled, { css } from 'styled-components';
+import { BoardInfo, BoardPinInfo } from '../types';
+import { FONT_STYLES } from '../styles/fonts/font';
+import { COLOR } from '../styles/color';
+
+interface PinImageProps {
+  idx: number;
+}
+
+interface BoardItemProps {
+  board: BoardInfo;
+}
+
+export default function BoardItem({ board }: BoardItemProps) {
+  return (
+    <StWrapper>
+      <StPinImageWrapper>
+        {board.boardList.map(
+          ({ id, pinImg }: BoardPinInfo) =>
+            id < 3 && (
+              <StImageWrapper idx={id}>
+                <StPinImage src={pinImg} key={id} />
+              </StImageWrapper>
+            ),
+        )}
+      </StPinImageWrapper>
+
+      <StPinInfoWrapper>
+        <StPinTitle>{board.title}</StPinTitle>
+        <StPinCount>핀 {board.boardList.length}개</StPinCount>
+        <StPinSavedTime>{board.savedTime}</StPinSavedTime>
+      </StPinInfoWrapper>
+    </StWrapper>
+  );
+}
+
+const StWrapper = styled.li`
+  display: flex;
+  flex-direction: column;
+`;
+const StPinImageWrapper = styled.article`
+  width: 173px;
+  height: 115px;
+  display: grid;
+  grid-template-columns: 115px 57px;
+  grid-template-rows: 57px 57px;
+  grid-gap: 0.5px 1px;
+  border-radius: 13px;
+  overflow: hidden;
+`;
+const StImageWrapper = styled.div<PinImageProps>`
```

Comment:
요 PinImageProps는 스컴포넌트만을 위한거니 굳이 위에 따로 선언해줄 필요 없이 이렇게 해줘도 될 것 같아요~~
```suggestion
const StImageWrapper = styled.div<{ idx: number }>`
```

Reply:
만약 지금 형태를 유지한다면 컴포넌트명+Props가 컨벤션이니 StImageWrapperProps가 맞는데 저는 그냥 위 형태로 해줘도 될 것 같습니다!